ppac-service-common/src/main/kotlin/com/levi9/ppac/service/api/domain/TrainerEntity.kt

```diff
@@ -0,0 +1,30 @@
+package com.levi9.ppac.service.api.domain
+
+import java.util.*
+import javax.persistence.Column
+import javax.persistence.Entity
+import javax.persistence.Id
+import javax.persistence.Table
+
+@Entity
+@Table(name = "TRAINERS")
+class TrainerEntity(
+
+    @Id
+    @Column(name = "ID")
+    var id: UUID,
+
+    @Column(name = "NAME", nullable = false)
+    var name: String,
+
+    @Column(name = "DESCRIPTION", nullable = false)
+    var description: String,
+
+    @Column(name = "LINKEDIN_URL", nullable = true)
+    var linkedinURL: String,
```

Comment:
If this field is nullable, I think It might not be necessary to be included in constructor